mythril/analysis/modules/external_calls.py

```diff
@@ -91,17 +91,16 @@ def _analyze_state(self, state: GlobalState):
                 state, constraints + state.world_state.constraints
             )
 
+            for tx in state.world_state.transaction_sequence:
+                if not isinstance(tx, ContractCreationTransaction):
+                    constraints.append(tx.caller == ACTORS.attacker)
             # Check whether we can also set the callee address
 
```

Comment:
What is the reason for the changes in this file?

Reply:
The contract provided pointed to two bugs, one is in calls and the other resulted in a buggy output after I fixed the call one